openlibrary/core/observations.py

```diff
@@ -1,26 +1,252 @@
 """Module for handling patron observation functionality"""
 
-import requests
+from collections import namedtuple
 
 from infogami import config
 from openlibrary import accounts
+
 from . import cache
+from . import db
+
+
+ObservationValue = namedtuple('ObservationValue', ['value_id', 'value', 'prev_value_id'])
```

Comment:
OPTIONAL...
```suggestion
ObservationValue = namedtuple('ObservationValue', 'value_id value prev_value_id')
```

Reply:
Leaving as-is.  Suggested change is, to my mind, a bit difficult to read.